### ISSUE TITLE
Path Segment Variable not work

### ISSUE THREAD

**Issue by jk2K** [danielgtaylor/aglio#193]
```
## get user info [GET /users/{uuid}]

+ Parameters

	+ uuid: `8E530903-2CEC-809B-6A35-BB91BBF602A9` (string) - unique identifier for user

+ Request (application/json)

+ Response 200 (application/json)

	+ Attributes
		+ data (MemberInfoBase) - user info
```
![image](https://cloud.githubusercontent.com/assets/4025839/11085830/94560188-8885-11e5-8b67-48473bbc24b4.png)

the `uuid` variable not render correctly

**Comment by danielgtaylor** [comment]
@jk2K what do you expect to see instead?

**Comment by jk2K** [comment]
`GET http://api.test.xmeise.com/users/8E530903-2CEC-809B-6A35-BB91BBF602A9`

use `8E530903-2CEC-809B-6A35-BB91BBF602A9` instead of `uuid` in example url

**Comment by danielgtaylor** [comment]
I think this is a good idea. I'll spend a little time looking into how it would work and if there are any weird edge cases to worry about.

**Comment by danielgtaylor** [comment]
@jk2K this should now be implemented in the olio-theme-1.6.1 release. Please take a look and let me know what you think. If no sample value is available, then the parameter name will continue to be used as it is today.

**Comment by jk2K** [comment]
work fine, thanks
